test/alternator/run

```diff
@@ -21,6 +21,22 @@ if '--aws' in sys.argv:
 import urllib3
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
+
+# If the first option is "--release", download that release (see
+# fetch_scylla.py for supported release numbers), and use that.
+# The downloaded Scylla will be cached in the directory build/<release>,
+# where <release> is the specific release downloaded - E.g., if the user
+# asks "--release 2022.1" and the downloaded release is 2022.1.9, it
+# will be stored in build/2022.1.9.
+if sys.argv[1] == '--release':
```

Comment:
On a side note, I see this script accepts --help switch, but it doesn't print options supported by this script, but the pytest options. So I assume that users of this script are expected to peek into the file to see what choices do they have. OK, it's not widely used script. That brings me to the next Q: why to make this --release positional argument: but I guess the UX doesn't matter that much here.

Reply:
Yes, the command-line processing here sucks - in the cql-pytest version I had a comment about this, and removed it here.
The original philosophy was that test/alternator/run wouldn't have its own options: it's just a wrapper to pytest and you pass to it all of pytest's options (e.g., which test to run, repeat, -v, -s, etc etc.). Now I wanted to add this "--release" option I just did the simplest and ugliest thing - require that it be the **first** option on the command line.
Trying to allow the "--release ..." option to be intertwined with all the pytest options is probably not worth the hassle, and if it is I can do it later.

Reply:
> Trying to allow the "--release ..." option to be intertwined with all the pytest options is probably not worth the hassle, and if it is I can do it later

I don't think it's worth the hassle as well, we can leave it as is.